src/store.js

```diff
@@ -68,7 +72,8 @@ class Store {
    * Binds an action type => handler
    */
   on(actionType, handler) {
-    this.__handlers = this.__handlers.set(actionType, handler)
+    console.assert(actionType, "Store.on called without actionType");
```

Comment:
1.  It doesn't seem like a standard https://developer.mozilla.org/en-US/docs/Web/API/console/assert
2.  I think the intended behavior here is to throw an error as this should be an invariant in the code.  Given that most people register stores at the initialization of your app throwing an error seems appropriate as you will always see it.